mantidproject/mantid
# Indirect Bayes - Post plot and save

The plotting and saving of the results workspace has been changed to after the execution of the algorithm.

**To test:**

Files to test with: \olympic\babylon5\Public\Louise McCann\Bayes
For each tab load relevant files and run algorithm. Check each plot option plots a suitable graph and that the appropriate workspaces have been saved in your save directory.
- [x] ResNorm
- [x] Quasi
- [x] Stretch

Fixes #17020.

Release notes will be updated once post plot and save has been completed for all Indirect tabs.

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
